tests/experimental/datasources/test_postgres_datasource.py

```diff
@@ -9,13 +9,13 @@
 from great_expectations.execution_engine import SqlAlchemyExecutionEngine
 from great_expectations.experimental.datasources.interfaces import (
     BatchRequest,
+    BatchRequestError,
```

Comment:
`BatchRequestError` was pulled up into the parent class so the import had to change.